CfdOF/Solve/CfdCaseWriterFoam.py

```diff
@@ -224,6 +224,9 @@ def processSystemSettings(self):
         system_settings['CasePath'] = self.case_folder
         system_settings['FoamPath'] = installation_path
         system_settings['TranslatedFoamPath'] = CfdTools.translatePath(installation_path)
+        system_settings['hostFileRequired'] = CfdTools.getSystemHostfileRequired(self.analysis_obj)
```

Comment:
I'm not sure that having a separate function adds anything here...couldn't we just say
system_settings['hostFileRequred'] = self.analysis_obj.UseHostfile
and
system_Settings['hostFileName'] = self.analysis_obj.HostfileName

Reply:
yeah sorry, thats better ...